app/views/users/index.html.erb

```diff
@@ -1,9 +1,13 @@
-<h1>All Users</h1>
-
-<ul>
-  <% @users.each do |user| %>
-    <li>
-      <%= link_to user.name, user_path(user) %>
-    </li>
-  <% end %>
-</ul>
+<div class="main-container">
+<% @users.each do |user| %>
+  <div class="users-wrapper">
+    <div class="img-cont">
+      <%= image_tag user.photo, :size => "70x85", alt: "user photo" %>
+    </div>
+    <div class="user-details">
+      <h1><%= link_to user.name, user, class: "link-button"  %></h1>
+      <p>Number of posts: <%= user.posts_counter %></p>
+    </div>
+  </div>
+<%end%>
```

Comment:
[OPTIONAL] It is a good practice to use partials in scenarios like this.

Reply:
Good day @fobadara I don't seem to see the issue on my side, I've deleted my repo and re-cloned it and ran the same command and it came back like this, I think I am noticing a trend because you are the 3rd code reviewer with a mac book that has struggled with my db

before recloning
![bandicam 2023-03-10 18-53-38-181](https://user-images.githubusercontent.com/90766123/224380599-97e68239-8525-4e6b-9b7c-d95f28e4e278.jpg)

after recloning
![bandicam 2023-03-10 19-15-13-552](https://user-images.githubusercontent.com/90766123/224380645-243f579f-6845-4cf1-933f-d3459a49185a.jpg)